Luke Smith Portfolio.css

```diff
@@ -0,0 +1,106 @@
+body {
+	margin: auto;
```

Comment:
Minor thing, but this should be aligned with the rest of the code in this section